nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -182,12 +185,15 @@ def __init__(
         activations_checkpoint_num_layers=1,
         activations_checkpoint_layers_per_pipeline=None,
         layernorm_epsilon=1e-5,
+        normalization='layernorm',
+        transformer_block_type='pre_ln',
```

Comment:
Not necessary to change, but should we consider making these values enums ? So that people dont do typos ?

Reply:
yeah, that will affect all models, maybe add in nemo refactor plan